style.css

```diff
@@ -5,76 +5,339 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
 
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1.2rem;
+  margin-left: 0.5rem;
+  justify-content: space-between;
+}
+
+.toolBar {
+  color: #6070ff;
+  font-size: 1.5rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.firstSec {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
+  width: 100%;
+}
+
+.intoHead {
+  color: #172b4d;
+  font-size: 40px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  line-height: 52px;
+  margin-bottom: 12px;
+  width: 80%;
+}
+
+.intoDes {
+  color: #344563;
+  font-size: 1.1rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  margin-bottom: 12px;
+  width: 85%;
+}
+
+.contact {
+  color: #7f8cff;
+  font-size: 0.8rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  line-height: 20px;
+  word-wrap: break-word;
+  margin-bottom: 12px;
+}
+
+.social {
+  margin-top: 0.5rem;
+}
+
+.social > a {
+  margin-right: 1rem;
+  font-size: 1.5rem;
+  color: #505f79;
+}
+
+.vt:hover {
+  border: 1px solid black;
+  padding: 2px;
+}
+
+.social i:hover {
+  color: black;
+}
+
+.works {
+  display: grid;
+  gap: 88px;
+  padding: 114px 24px;
+}
+
+.cards h2 {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 12px 0 0 0;
+}
+
+.brief-list {
+  font-family: 'Poppins', sans-serif;
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin: 12px 0 0 0;
+}
+
+.self-intro {
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.self-intro p {
+  text-align: justify;
+}
```

Comment:
- [ ] In this code your h2 tag, span tag and p tage spacing are very close and for your paragraph you shuoul give specified width and height and also you need to same font size of language buttons for all the cards of work section to look good and check out this images.

**Your Design**

![image](https://github.com/mioOleen/Portfolio-1/assets/118582316/cc8efe1b-7f1b-40af-a3e3-562681d3aebc)

**Figma Design**

![image](https://github.com/mioOleen/Portfolio-1/assets/118582316/c6cbdc3a-a233-4f0b-b9e0-42d5b7e6d173)